proposals/accepted/051-ghc-base-libraries.rst

```diff
@@ -0,0 +1,316 @@
+================
+GHC's base libraries
+================
+----------
+Combining stability with innovation
+----------
+
+:Date: June 2023
+:Authors:
+  John Ericson,
+  Ben Gamari,
+  Adam Gundry,
+  Andrew Lelechenko,
+  Julian Ospald,
+  Simon Peyton Jones
+
+.. sectnum::
+.. contents::
+
```

Comment:
Introduction
=========
This document describes an plan agreed between the GHC Team and the Core Libraries Committee, saying how we plan to work in partnership to reconcile the goals of innovation and stability for GHC and its ecosystem.

Formally, then, it is not so much a proposal as a record of an outcome.  We are, nevertheless, using the Haskell Foundation Technical Working Group propsals repo, so that we can have a permanent  public record of how we plan to work, so that others can comment, and so that the document can be polished to add clarity where necessary.

Reply:
```suggestion
Introduction
=========

This document describes an plan agreed between the GHC Team and the Core Libraries Committee, saying how we plan to work in partnership to reconcile the goals of innovation and stability for GHC and its ecosystem.

Formally, then, it is not so much a proposal as a record of an outcome.
We are, nevertheless, using the Haskell Foundation Technical Working Group proposals repo, so that we can have a permanent public record of how we plan to work, so that others can comment, and so that the document can be polished to add clarity where necessary.
```